ANS_JS_template.tex

```diff
@@ -93,92 +93,92 @@
 
-Renewable sources of energy and nuclear power both have the capacity to play an important role in mitigating climate change \cite {IPCC}. Due to the growth in variable energy generating sources, particularly wind and solar, grid flexibility has become a key driver in power system development \cite {Denholm2011}. Power system flexibility is defined by the Electric Power Research Institute (EPRI) as the "ability to adapt to dynamic and changing conditions, for example, balancing supply and demand by the hour or minute, or deploying new generation and transmission resources over a period of years \cite{EPRI2016}." Power system flexibility differs from load-following.  Load following is described as base-load power generator reductions \cite{Bragg-Sitton2014} or a power plant that adjusts its power output as demand for electricity fluctuates \cite{Masters2004}. In the case of a NRHES, the nuclear power plant (NPP) will continue to run at full power output, allocating more or less energy to the grid depending on demand. Load following generally does not include following the predictable seasonal differences in electricity demand.  Optimally, a nuclear renewable hybrid energy system will be able to respond to fluctuations in both seasonal and hourly demand. A NRHES combines power system flexibility with load following by adapting to dynamic conditions by changing the output of electricity to the grid.
+Renewable sources of energy and nuclear power both have the capacity to play an important role in mitigating climate change \cite {IPCC}. Due to the growth in variable energy generating sources, particularly wind and solar, grid flexibility has become a key driver in power system development \cite {Denholm2011}. Power system flexibility is defined by the Electric Power Research Institute (EPRI) as the "ability to adapt to dynamic and changing conditions, for example, balancing supply and demand by the hour or minute, or deploying new generation and transmission resources over a period of years \cite{EPRI2016}." Power system flexibility differs from load-following.  Load following is described as base-load power generator reductions \cite{Bragg-Sitton2014} or a power plant that adjusts its power output as demand for electricity fluctuates \cite{Masters2004}. Load following generally does not include following the predictable seasonal differences in electricity demand, but focuses more on the daily shifts. In the case of a NRHES, the nuclear power plant (NPP) will continue to run at full power output, allocating more or less electricity to the grid depending on demand.   Optimally, a NRHES will be able to respond to fluctuations in both seasonal and hourly demand. A NRHES combines power system flexibility with load following, adapting to dynamic conditions by changing the output of electricity to the grid.
 
```

Comment:
Flipped two sentences:  Load following generally does not include following the predictable seasonal differences in electricity demand, but focuses more on the daily shifts. In the case of a NRHES, the nuclear power plant (NPP) will continue to run at full power output, allocating more or less electricity to the grid depending on demand.